local/src/main/java/com/project/local/model/LocalDao.kt

```diff
@@ -11,7 +11,7 @@ import io.reactivex.Maybe
 interface LocalDao {
 
     @Query("SELECT * FROM list")
-    fun getMovieList(): Maybe<LocalItem>
+    fun getList(): Maybe<LocalItem>
```

Comment:
해당 함수가 어떤 일을 담당하는지 모르겠어요.
로컬 db에 있는 유저목록을 가져올것인가요 ? 아니면 내부에서 저장한 여행 목록을 불러올것인가요 ?
```suggestion
fun get...List(): Maybe<*> {
    ...
}
```
스타 프로젝션은 임의로 넣어둔 값 입니다. ...은 유저목록이면 `User` 여행 목록이면 `travel` 로 사용했으면 좋겠어요 ~

Reply:
지금은 아직 기획이 다 안나온 상태에서 미리 구현했기 때문에 어떠한 리스트를 가져오는게 명확하진 않은 상태로 미리 구현하였습니다~! 로컬은 필요없을 수도 있단 생각도 드네요~